ceph-release-rpm/config/definitions/ceph-release-rpm.yml

```diff
@@ -9,7 +9,7 @@
     parameters:
       - string:
           name: RELEASE
-          default: firefly
+          default: pacific
```

Comment:
so, after the change, the chacra URL will be `ceph-release/pacific/HEAD/centos/8/noarch` for non-test centos8 builds. makes sense to me.